src/OpenApi/test/OpenApiRouteHandlerBuilderExtensionTests.cs

```diff
@@ -136,8 +136,16 @@ public void WithOpenApi_WorksWithMapGroupAndEndpointAnnotations()
         var groupDataSource = Assert.Single(builder.DataSources);
         var endpoint = Assert.Single(groupDataSource.Endpoints);
         var operation = endpoint.Metadata.GetMetadata<OpenApiOperation>();
+
         Assert.NotNull(operation);
-        Assert.Equal("201", operation.Responses.Keys.SingleOrDefault());
+        Assert.Equal(2, operation.Responses.Count);
+
+        var defaultOperation = operation.Responses["200"];
+        Assert.True(defaultOperation.Content.ContainsKey("text/plain"));
+
+        var annotatedOperation = operation.Responses["201"];
+        // Produces doesn't special case string??
```

Comment:
What do you mean by this comment?

Reply:
I was just surprised when updating this test that `.Produces<string>(201);` creates "application/json" metadata. I would have expected "text\plain", but this is existing behavior. You can still explicitly set the content-type if you want.

Reply:
Wait it does? That seems like a bug.

Reply:
Yep, this has been behavior for a while. The `Produces` extension method sets the type to `application/json` by default unless explicitly defined.

We had some extensive conversations about this when we were building out the defaults and decided that setting the default to `application/json` would be sensible for most minimal cases.

Reply:
#43979. If we agree it's important, I'll try to get a PR out by tomorrow before branching. I'm also going to open an API proposal about making `Produces` and `Accepts` work with groups. I think we should probably just retarget these from `RouteHandlerBuilder` to `IEndopintConventionBuilder`, but we can consider `RouteGroupBuilder`-specific overloads although that's less extensible.